src/token_cache.rs

```diff
@@ -59,7 +59,26 @@ pub enum RegistryOperation {
     Pull,
 }
 
-type CacheType = BTreeMap<(String, String, RegistryOperation), (RegistryTokenType, u64)>;
+// Types to allow better naming
+type Registry = String;
```

Comment:
Nit: I don't really like aliased types like this because they lead to confusion down the line (an extra mental hop to figure out it is a string). I can get behind them if defining a complex type, but if they are just used in a struct like they are below, it is pretty clear what it is for due to the field name

Reply:
solved